File: tmp-mei/fgen4255027576711118260/base/Infos_0.java
package base;
public interface Infos_0{
Infos_0 $self = new Infos_0Impl();
base.Info_0 list$imm(base.List_1 list_m$);

base.Info_0 msg$imm(rt.Str msg_m$);

base.Info_0 map$imm(base.LinkedHashMap_2 map_m$);
static base.Info_0 msg$imm$fun(rt.Str msg_m$, base.Infos_0 $this) {
  return ((base.Info_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Info_0) null;
}})
);
}

static base.Info_0 list$imm$fun(base.List_1 list_m$, base.Infos_0 $this) {
  return ((base.Info_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Info_0) null;
}})
);
}

static base.Info_0 map$imm$fun(base.LinkedHashMap_2 map_m$, base.Infos_0 $this) {
  return ((base.Info_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Info_0) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/Infos_0.js
import * as rt from "../rt/main.js";
export class base$$Infos_0 {
  static $self = new base$$Infos_0();

  list$imm(list_m$) {
  return base$$Infos_0.list$imm$fun(list_m$, this);
}


msg$imm(msg_m$) {
  return base$$Infos_0.msg$imm$fun(msg_m$, this);
}


map$imm(map_m$) {
  return base$$Infos_0.map$imm$fun(map_m$, this);
}

  static msg$imm$fun(msg_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static list$imm$fun(list_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static map$imm$fun(map_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
